cmd/admin-handlers.go

```diff
@@ -245,7 +245,7 @@ func (a adminAPIHandlers) ServerInfoHandler(w http.ResponseWriter, r *http.Reque
 			// Initialize server info at index
 			reply[idx] = ServerInfo{Addr: peer.addr}
 
-			serverInfoData, err := peer.cmdRunner.ServerInfoData()
+			serverInfoData, err := peer.cmdRunner.ServerInfo()
```

Comment:
@balamurugana it is better than you avoid naming changes, I expect to only see that HTTP based RPC support in this PR. If you wish to do that you can do that in next iteration.